scalastyle-config.xml

```diff
@@ -109,7 +109,7 @@
  </check>
  <check level="warning" class="org.scalastyle.scalariform.NumberOfMethodsInTypeChecker" enabled="true">
   <parameters>
-   <parameter name="maxMethods"><![CDATA[30]]></parameter>
+   <parameter name="maxMethods"><![CDATA[35]]></parameter>
```

Comment:
If this is for `GetManyDocumentsQueryBuilder`, let's add a todo to change it back when the deprecated methods are removed.